Bottom: f6f2d24a4ca9fdddabfaf4e99e072d01586f16e4
Top:    f0c180d833d20dbe1c788452b8e743b3b0ef6815
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 22:08:18 +0100

Refresh of misc-c-remove-unused-delta-arg

---

diff --git a/src/action.c b/src/action.c
index d6e1d89..d892429 100644
--- a/src/action.c
+++ b/src/action.c
@@ -565,7 +565,7 @@ FinishStroke (void)
 		   log (2.0));
 	    SetZoom (z);
 
-	    CenterDisplay (x, y, false);
+	    CenterDisplay (x, y);
 	    break;
 	  }
 
@@ -3836,8 +3836,7 @@ ActionAddRats (int argc, char **argv, int x, int y)
 	      DrawRat (shorty);
 	      Draw ();
 	      CenterDisplay ((shorty->Point2.X + shorty->Point1.X) / 2,
-			     (shorty->Point2.Y + shorty->Point1.Y) / 2,
-			     false);
+			     (shorty->Point2.Y + shorty->Point1.Y) / 2);
 	    }
 	  break;
 	}
@@ -5957,7 +5956,7 @@ ActionNew (int argc, char **argv, int x, int y)
       ResetStackAndVisibility ();
       CreateDefaultFont ();
       SetCrosshairRange (0, 0, PCB->MaxWidth, PCB->MaxHeight);
-      CenterDisplay (PCB->MaxWidth / 2, PCB->MaxHeight / 2, false);
+      CenterDisplay (PCB->MaxWidth / 2, PCB->MaxHeight / 2);
       Redraw ();
 
       hid_action ("PCBChanged");
diff --git a/src/find.c b/src/find.c
index 92c8b35..2f9053d 100644
--- a/src/find.c
+++ b/src/find.c
@@ -4532,7 +4532,7 @@ GotoError (void)
                              (PCB->Data, (LayerTypePtr) thing_ptr1), true,
                              true);
     }
-  CenterDisplay (X, Y, false);
+  CenterDisplay (X, Y);
 }
 
 void
diff --git a/src/hid/gtk/gui-drc-window.c b/src/hid/gtk/gui-drc-window.c
index 674e495..6d50d16 100644
--- a/src/hid/gtk/gui-drc-window.c
+++ b/src/hid/gtk/gui-drc-window.c
@@ -260,7 +260,7 @@ row_activated_cb (GtkTreeView *view, GtkTreePath *path,
   if (violation == NULL)
     return;
 
-  CenterDisplay (violation->x_coord, violation->y_coord, false);
+  CenterDisplay (violation->x_coord, violation->y_coord);
 }
 
 
diff --git a/src/misc.c b/src/misc.c
index ce2ab4a..befeeed 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -769,17 +769,12 @@ GetDataBoundingBox (DataTypePtr Data)
  * coordinates
  */
 void
-CenterDisplay (LocationType X, LocationType Y, bool Delta)
+CenterDisplay (LocationType X, LocationType Y)
 {
   double save_grid = PCB->Grid;
   PCB->Grid = 1;
-  if (Delta)
-    MoveCrosshairRelative (X, Y);
-  else
-    {
-      if (MoveCrosshairAbsolute (X, Y))
-        notify_crosshair_change (true);
-    }
+  if (MoveCrosshairAbsolute (X, Y))
+    notify_crosshair_change (true);
   gui->set_crosshair (Crosshair.X, Crosshair.Y, HID_SC_WARP_POINTER);
   PCB->Grid = save_grid;
 }
diff --git a/src/misc.h b/src/misc.h
index 45d6e52..7926460 100644
--- a/src/misc.h
+++ b/src/misc.h
@@ -59,7 +59,7 @@ bool IsLayerNumEmpty (int);
 bool IsLayerGroupEmpty (int);
 bool IsPasteEmpty (int);
 BoxTypePtr GetDataBoundingBox (DataTypePtr);
-void CenterDisplay (LocationType, LocationType, bool);
+void CenterDisplay (LocationType, LocationType);
 void SetFontInfo (FontTypePtr);
 int ParseGroupString (char *, LayerGroupTypePtr, int /* LayerN */);
 int ParseRouteString (char *, RouteStyleTypePtr, const char *);
